x11-misc/dunst/dunst-1.11.0-r1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd toolchain-funcs
+
+DESCRIPTION="Lightweight replacement for common notification daemons"
+HOMEPAGE="https://dunst-project.org/ https://github.com/dunst-project/dunst"
+SRC_URI="https://github.com/dunst-project/dunst/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~x86"
+IUSE="+X wayland +dunstify +completions +xdg"
+
+DEPEND="
+	dev-libs/glib:2
+	sys-apps/dbus
+	x11-libs/cairo[glib]
```

Comment:
Instead adding ```x11-libs/cairo[X]``` to ```X? (``` you could do instead: ```x11-libs/cairo[X?,glib]```, so you don't have to add twice the same dep with different use flags.